src/main/java/swervelib/encoders/SparkMaxEncoderSwerve.java

```diff
@@ -52,19 +42,24 @@ public SparkMaxEncoderSwerve(SwerveMotor motor, int conversionFactor)
         "Encoders",
         "Failure to set Absolute Encoder Offset",
         Alert.AlertType.WARNING_TRACE);
```

Comment:
I would initialize these alerts at the point of their declaration, to further strengthen the code and make it difficult to hit the error we were hitting...

Reply:
this just makes the alert component doesn't actually throw the error

Reply:
ask alex more about this